src/assets/CollectiblesController.test.ts

```diff
@@ -181,387 +187,491 @@ describe('CollectiblesController', () => {
+      expect(collectiblesController.state.ignoredCollectibles).toHaveLength(1);
     });
-  });
 
-  it('should not add collectibles with no contract information when auto detecting', async () => {
-    await collectiblesController.addCollectible(
-      '0x6EbeAf8e8E946F0716E6533A6f2cefc83f60e8Ab',
-      '123',
-      undefined,
-      true,
-    );
-    expect(collectiblesController.state.collectibles).toStrictEqual([]);
-    expect(collectiblesController.state.collectibleContracts).toStrictEqual([]);
-    await collectiblesController.addCollectible(
-      ERC721_KUDOSADDRESS,
-      '1203',
-      undefined,
-      true,
-    );
-
-    expect(collectiblesController.state.collectibles).toStrictEqual([
-      {
-        address: ERC721_KUDOSADDRESS,
-        description: 'Kudos Description',
-        imageOriginal: 'Kudos url',
-        name: 'Kudos Name',
-        image: null,
-        standard: 'ERC721',
-        tokenId: '1203',
-        collectionImage: 'collection.url',
-        collectionName: 'Collection Name',
-      },
-    ]);
+    it('should add collectible with metadata hosted in IPFS', async () => {
+      assetsContract.configure({ provider: MAINNET_PROVIDER });
+      await collectiblesController.addCollectible(
+        ERC1155_DEPRESSIONIST_ADDRESS,
+        ERC1155_DEPRESSIONIST_ID,
+      );
 
-    expect(collectiblesController.state.collectibleContracts).toStrictEqual([
-      {
-        address: ERC721_KUDOSADDRESS,
-        description: 'Kudos Description',
-        logo: 'Kudos url',
-        name: 'Kudos',
-        symbol: 'KDO',
-        totalSupply: 10,
-      },
-    ]);
-  });
+      expect(
+        collectiblesController.state.collectibleContracts[0],
+      ).toStrictEqual({
+        address: '0x18E8E76aeB9E2d9FA2A2b88DD9CF3C8ED45c3660',
+        name: "Maltjik.jpg's Depressionists",
+        symbol: 'DPNS',
+      });
 
-  it('should remove collectible and collectible contract', async () => {
-    await collectiblesController.addCollectible('0x01', '1', {
-      name: 'name',
-      image: 'image',
-      description: 'description',
-      standard: 'standard',
+      expect(collectiblesController.state.collectibles[0]).toStrictEqual({
+        address: '0x18E8E76aeB9E2d9FA2A2b88DD9CF3C8ED45c3660',
+        tokenId: '36',
+        image: 'image',
+        name: 'name',
+        description: 'description',
+        standard: 'ERC721',
+      });
     });
-    collectiblesController.removeCollectible('0x01', '1');
-    expect(collectiblesController.state.collectibles).toHaveLength(0);
-    expect(collectiblesController.state.collectibleContracts).toHaveLength(0);
   });
 
-  it('should not remove collectible contract if collectible still exists', async () => {
-    await collectiblesController.addCollectible('0x01', '1', {
-      name: 'name',
-      image: 'image',
-      description: 'description',
-      standard: 'standard',
+  describe('addCollectibleVerifyOwnership', () => {
```

Comment:
TLDR: I reorganized the tests such that each major method has its own describe block.